playbook.yml

```diff
@@ -30,7 +30,7 @@
         name: sysbench
       vars:
         sysbench : "{{ workload.args }}"
-      when: workload.name == "sysbench" and workload.args.enabled is defined and workload.args.enabled
+      when: workload.name == "sysbench" and workload.args.client > 0 and workload.args.enabled
```

Comment:
Any specific reason for this change?

Reply:
reverting this , AFAIR it was for experimental puposes